include/seqan3/range/view/pairwise_combine.hpp

```diff
@@ -0,0 +1,595 @@
+ * size of the source range. The reference type of this range is a tuple over the reference type of the source range.
+ * Constness will not be propagated to the reference types of the source range.
+ *
+ * ### View properties
+ *
+ * | range concepts and reference_t  | `urng_t` (underlying range type)      | `rrng_t` (returned range type)                                       |
+ * |---------------------------------|:-------------------------------------:|:--------------------------------------------------------------------:|
+ * | std::ranges::InputRange         |                                       | *undefined*                                                          |
+ * | std::ranges::ForwardRange       | *required*                            | *preserved*                                                          |
+ * | std::ranges::BidirectionalRange |                                       | *preserved*                                                          |
+ * | std::ranges::RandomAccessRange  |                                       | *preserved*                                                          |
+ * | std::ranges::ContiguousRange    |                                       | *lost*                                                               |
+ * |                                 |                                       |                                                                      |
+ * | std::ranges::ViewableRange      | *required*                            | *guaranteed*                                                         |
+ * | std::ranges::View               |                                       | *guaranteed*                                                         |
+ * | std::ranges::SizedRange         |                                       | *guaranteed* iff urng_t models std::RandomAccessRange                |
+ * | std::ranges::CommonRange        |                                       | *guaranteed*                                                         |
+ * | std::ranges::OutputRange        |                                       | *lost*                                                               |
+ * | seqan3::const_iterable_concept  |                                       | *preserved*                                                          |
+ * |                                 |                                       |                                                                      |
+ * | seqan3::reference_t             |                                       | std::tuple<seqan3::reference_t<urng_t>, seqan3::reference_t<urng_t>> |
+ *
+ * See the \link view view submodule documentation \endlink for detailed descriptions of the view properties.
+ *
+ * ### Thread safety
+ *
+ * Concurrent access to this view, e.g. while iterating over it, is not thread-safe and must be protected externally.
```

Comment:
This sounds like it is not safe to iterate over the view concurrently, but it is, or not? If not, you can't mark this as preserving const-iterable.

Reply:
Copy'n'paste error.